data/catalogs/deltares_data.yml

```diff
@@ -1196,8 +1217,13 @@ hydro_reservoirs:
     category: surface water
     notes: GRanD.v1.1_HydroLAKES.v10_JRC.2016
     source_author: Alessia Matano
```

Comment:
For hydro_reservoirs,  this is actually GranD v1.1 as main: https://sedac.ciesin.columbia.edu/data/set/grand-v1-dams-rev01 
So maybe swith all source info and author to it.
Then in processing_notes: merge attributes to corresponding reservoirs in HydroLakes v10 and add statistics computed from JRC 2016 timeseries by Deltares (Alessia Matano).
Personal note: the JRC statistics we actually do not use (I think there was an error in the script?) and compute them again on the fly in hydromt-wflow from JRC or even global water watch now.